tests/service_controller_test.go

```diff
@@ -0,0 +1,48 @@
+package tests
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	"kubevirt.io/ssp-operator/controllers"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func getSspMetricsService() (*v1.Service, error) {
+	service := controllers.ServiceObject(strategy.GetSSPDeploymentNameSpace())
+	err := apiClient.Get(ctx, client.ObjectKeyFromObject(service), service)
+	return service, err
+}
+
+var _ = Describe("Service Controller", func() {
+	var (
+		serviceErr error
+		service    *v1.Service
+	)
+	BeforeEach(func() {
+		waitUntilDeployed()
+
+		service, serviceErr = getSspMetricsService()
```

Comment:
Can you move this call to the `It` test cases? It makes more sense to be there, and the error can be handled right there.